client/my-sites/plugins/plugin-ratings/index.jsx

```diff
@@ -118,7 +114,7 @@ const PluginRatings = createReactClass( {
 			return null;
 		}
 
-		const tierViews = this.ratingTiers.map( ( tierLevel ) => this.renderRatingTier( tierLevel ) );
+		const tierViews = ratingTiers.map( ( tierLevel ) => this.renderRatingTier( tierLevel ) );
```

Comment:
What about using point-free style here `ratingTiers.map( this.renderRatingTier )`?

Nit: not related to your PR but isn't _tier_ already expressing a certain stage and _tierLevel_ therefore would be redundant? (I'm not a native english speaker so I may be wrong)

Reply:
Good call! I've changed this as you suggested in https://github.com/Automattic/wp-calypso/pull/48170/commits/1bc584b16df51703de027253e4c0e8599c0ed5c3

Reply:
There seems to be something wrong with my suggestion. Now when I click on a plugin from the list I get `TypeError: Cannot read property 'props' of undefined` in `renderRatingTier`. Do we have to pass `this` as `thisArg` to the [map](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Array/map) function?

Reply:
Yep, it had to be an arrow function in order to be automatically bound to `this`.

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/48170/commits/c90d3e40c56e4dbea2047377f05223ea508f268f.

Fun fact: the one change I didn't test ended up broken 🙈